pyrasterframes/src/main/python/tests/PyRasterFramesTests.py

```diff
@@ -139,6 +141,22 @@ def test_tile_udt_serialization(self):
             long_trip = df.first()["tile"]
             self.assertEqual(long_trip, a_tile)
 
+    def test_masked_deser(self):
```

Comment:
Not sure I understand what this is testing...

Reply:
basically just testing that a Tile that gets deserialized has the expected masking on it.

Reply:
Basically is the same thing as my commented out block at RasterFunctionsTest.py lines 282 on.... paranoia